docs/reference/indices/flush.asciidoc

```diff
@@ -101,7 +101,8 @@ which returns something similar to:
                      "translog_generation" : "2",
                      "max_seq_no" : "-1",
                      "sync_id" : "AVvFY-071siAOuFGEO9P", <1>
-                     "max_unsafe_auto_id_timestamp" : "-1"
+                     "max_unsafe_auto_id_timestamp" : "-1",
+                     "soft_deletes_min_retained_seq_no": "-1"
```

Comment:
can we just call it `min_retained_seq_no` I don't see a reason to exploit the impl detail of soft_deletes